### PR TITLE
S3 task two functionality to edit review

### PR DESCRIPTION
**Pull request by EleanorRoberts**

### CODE REVIEW

File: collectionfunctions.php [hunk 1]
```diff
@@ -1,13 +1,13 @@
 <?php
 
-/** Displays each pokemon and stats
+/** Displays each Pokémon and stats
```
Comment (jesshamby) [inline collectionfunctions.php:3]: Love the correction here 😍

File: collectionfunctions.php [hunk 2]
```diff
@@ -1,13 +1,13 @@
...
  * @param array $pokemon
  * @return String
  */
 function displayPoke(Array $pokemon): String {
     $output = "";
     foreach ($pokemon as $poke) {
-        $output .= "<div class='pokeCard'>";
+        $output .= "<div class='pokeCard'";
```
Comment (jesshamby) [inline collectionfunctions.php:10]: Should this > have been removed?

File: collectionfunctions.php [hunk 3]
```diff
@@ -83,7 +93,36 @@ function checkNew(PDO $database) {
...
             }
         } else {
-            echo "<div class='error'>Unable to add Pokemon to the collection</div>";
+            echo "<div class='error'>Unable to add Pokémon to the collection</div>";
+        }
+    }
+}
+
+function editPoke(PDO $database) {
+    if (isset($_POST['name'], $_POST['type1'], $_POST['hp'], $_POST['attack'], $_POST['defense'], $_POST['spAttack'], $_POST['spDefense'], $_POST['speed'])) {
```
Comment (jesshamby) [inline collectionfunctions.php]: Could potentially put these 2 lines in a function so you aren't repeating yourself (also in the checkNew function above)? :)

File: editpokemon.php [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+<head>
+    <meta charset="UTF-8" />
+    <title>Edit a Pokemon</title>
+    <?php echo addStyle(); ?>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+</head>
+<body>
+<h1>Edit Your Pokemonz</h1>
+<form method="post">
+    <label for="name">Name <input type="text" name="name" value="<?php echo $poke['name'] ?>" required /></label>
```
Comment (jesshamby) [inline editpokemon.php]: Could maybe indent your input tag on a new line to make this more readable :)

**Review by jesshamby — COMMENTED** [review]
Looks really good! Love how you've got everything in functions! Just a few small changes :)

File: editpokemon.php (hunk 4 above)
Comment (EleanorRoberts) [reply editpokemon.php]: Done :)

File: collectionfunctions.php (hunk 3 above)
Comment (EleanorRoberts) [reply collectionfunctions.php]: Amended to a variable :)

**Pull request merged**